docs/reference/transform/transforms-at-scale.asciidoc

```diff
@@ -0,0 +1,197 @@
+increasing its shard count. {transforms-cap} use one shard by default when 
+creating the destination index. To override the index settings, create the 
+destination index before starting the {transform}.
+
+Hint: Use the <<preview-transform>> to check the settings that the {transform} 
+would use to create the destination index. You can copy and adjust these in 
+order to create the destination index prior to starting the {transform}.
+
+
+[discrete]
+[[optimize-shading-strategy]]
+== 5. Optimize the sharding strategy for the source index (search)
+
+There is no one-size-fits-all sharding strategy. A strategy that works in one 
+environment may not scale in another. A good sharding strategy must account for 
+your infrastructure, use case, and performance expectations.
+
+Too few shards may mean that the benefits of distributing the workload cannot be 
+realised; however too many shards may impact your cluster health. To learn more 
+about sizing your shards, read this <<size-your-shards,guide>>.
+
+
+[discrete]
+[[disable-source-dest]]
+== 6. Disable the `_source` field on the destination index (storage)
+
+The <<mapping-source-field>> contains the original JSON document body that was 
+passed at index time. The `_source` field itself is not indexed (and thus is not 
+searchable), but it is still stored in the index and incurs a storage overhead. 
+Consider disabling `_source` to save storage space if you have a large 
+destination index. Disabling `_source` is only possible during index creation.
+
+NOTE: When the `_source` field is disabled, a number of features are not 
+supported. Consult <<disable-source-field>> to understand the consequences 
+before disabling it.
+
+
+[discrete]
+[[tune-max-page-search-size]]
+== 7. Tune `max_page_search_size` (search)
+
+The `max_page_search_size` parameter defines the number of buckets that are 
+returned for each search request. The default value is 500. If you increase this 
+value, you get better throughput at the cost of higher latency and memory usage.
+
+The ideal value of this parameter is highly dependent on your use case. If your 
+{transform} executes memory-intensive aggregations – for example, cardinality or 
+percentiles – then increasing `max_page_search_size` requires more available 
+memory. A circuit breaker exception will occur if memory limits are exceeded.
+
+
+[discrete]
+[[indexed-fields-in-source]]
+== 8. Use indexed fields in your source indices (search)
+
+Runtime fields and scripted fields are not indexed fields, and their values are 
+only extracted or computed at search time. While these fields provide 
+flexibility in how you access your data, they increase performance costs at 
+search time. If {transform} performance using runtime fields or scripted fields 
+is a concern, you may wish to consider using indexed fields instead. For 
+performance reasons, we do not recommend using a runtime field as the time field 
+used for syncing a {ctransform}. 
```

Comment:
Minor nit, but can we avoid putting "use" twice in this sentence? Maybe something like this:
```suggestion
that synchs a {ctransform}. 
```